docs/features/techdocs/using-cloud-storage.md

```diff
@@ -265,3 +265,60 @@ techdocs:
 
 Your Backstage app is now ready to use AWS S3 for TechDocs, to store the static
 generated documentation files.
+
+## Configuring Azure Storage Bucket with TechDocs
+
+Follow the
+[official Azure Storage documentation](https://docs.microsoft.com/pt-br/javascript/api/@azure/storage-blob/?view=azure-node-latest)
+for the latest instructions on the following steps involving Azure Storage.
+
+**1. Set `techdocs.publisher.type` config in your `app-config.yaml`**
+
+Set `techdocs.publisher.type` to `'azureStorage'`.
+
+```yaml
+techdocs:
+  publisher:
+    type: 'azureStorage'
+```
+
+**2. Service account credentials**
+
+To get credentials access Azure Portal and go to Settings > Access Keys, and get
+your Storage account name and Primary Key.
+
+```yaml
+techdocs:
+  publisher:
+    type: 'azureStorage'
+    azureStorage:
+      credentials:
+        account: 'account'
+        accountKey: 'accountKey'
+```
+
+**3. Azure Storage Bucket**
+
+Create a dedicated bucket for TechDocs sites. techdocs-backend will publish
```

Comment:
Curious - does Azure call these buckets?  i thought it used different terminology, but for SURE I won't claim to be an expert here :) thought I'd ask

Reply:
I will change all mentions from bucket to container